test_stream_zip.py

```diff
@@ -1059,7 +1091,7 @@ def test_unzip_modification_time(method, timezone, modified_at):
 
         subprocess.run(['unzip', f'{d}/test.zip', '-d', d], env={'TZ': timezone})
 
-        assert os.path.getmtime('my_file') == int(modified_at.timestamp())
+        assert os.path.getmtime('my_file') == int(expected_modified_at.timestamp())
```

Comment:
The test failures seem to be caused by the `unzip` being run with the given timezone variable, but `expected_modified_at.timestamp()` being interpreted as local time when it has no `tzinfo` set.